Блог.html

```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+<title>Your business</title>
+</head>
+<body>
+<header>
+<div>
+<h1>Press</h1>
+<p style="color:grey;">Получайте еженедельные статьи в свой почтовый ящик о том, как развивать свой бизнес</p>
+<form action = "">
+  <input type="text" placeholder="Enter your email" name="email" required>
+  <button type="submit" >Сообщите мне</button>
+</form>
+</div>
+</header>
+
+<article class="first-area">
+<header>
+  <p> <time 2020-3-16></time>Mar 16, 2020</p>
```

Comment:
не корректно
сделать так, и заменить везде
```suggestion
  <time>Mar 16, 2020</time>
```